webpack.config.js

```diff
@@ -7,10 +7,10 @@ module.exports = {
   entry: path.join(__dirname, 'assets/src/index'),
   output: {
     path: path.join(__dirname, 'assets/dist'),
-    filename: '[name]-[hash].js'
+    filename: '[name]-[fullhash].js'
```

Comment:
Is this change part of upgrade to new webpack version?

Reply:
Yes, there was an error or deprecation warning, I forget which